docs/api-reference/index.rst

```diff
@@ -4,6 +4,50 @@ API Reference
 Warehouse has several API endpoints. See :doc:`../application` for the
 parts of Warehouse that generate them.
 
+.. contents:: :local:
+
+API Policies
+------------
+
+Please be aware of the following PyPI API Policies:
+
+Caching
+~~~~~~~
+
+All API requests are cached. Requests to the JSON, RSS or Legacy APIs are
+cached by our CDN provider. You can determine if you've hit the cache based on
+the ``X-Cache`` and ``X-Cache-Hist`` headers in the response.
+
+All API requests also provide an ``ETag`` header. If you're making a lot of
+repeated requests, please ensure your API consumer will respect this header to
+determine whether to actually repeat a request or not.
+
+Rate Limiting
+~~~~~~~~~~~~~
+
+Due to the heavy caching and CDN use, there is currently no rate limiting of
+PyPI APIs.
```

Comment:
*At the edge. Requests that hit our backends (uploads, XMLRPC) may be rate limited.